docs/RC/RC.OnReverseAppsAllowing/index.md

```diff
@@ -0,0 +1,61 @@
+## RC.OnReverseAppsAllowing
```

Comment:
Also "OnReverseAppsAllowing" is not a very well named RPC. I believe this should be changed to something less nonsensical. I would provide a suggestion but the purpose description below isn't very clear on what the function of this rpc is.